test/simulator/main/org/apache/cassandra/simulator/paxos/AbstractPairOfSequencesPaxosSimulation.java

```diff
@@ -209,6 +209,8 @@ public ActionPlan plan()
             public Action get()
             {
                 int[] primaryKeyIndex = consume(simulated.random, available);
+                if (primaryKeyIndex == null)
```

Comment:
this should fix the issue where "consume" is empty, and by returning `Actions.empty` we don't halt the simulation